plugins/BEdita/Core/src/Utility/ProjectModel.php

```diff
@@ -140,11 +153,19 @@ protected static function properties(): array
             ->find('type', ['dynamic'])
             ->all()
             ->each(function (EntityInterface $row) {
-                unset($row['id']);
-                unset($row['created']);
-                unset($row['modified']);
-                unset($row['label']);
-                unset($row['is_static']);
+                $hidden = [
+                    'id',
+                    'created',
+                    'modified',
+                    'label',
+                    'is_static',
+                    'object_type_name',
+                    'property_type_name',
+                ];
+                $row->setAccess(['object', 'property'], true);
```

Comment:
We can remove  `$row->setAccess(['object', 'property'], true);` because we set a single property at the time.